streaming_form_data/parser.py

```diff
@@ -8,11 +8,10 @@ class ParseFailedException(Exception):
 
 
 def parse_content_boundary(headers):
-    content_type_header = 'Content-Type'
-
-    content_type = headers.get(content_type_header) or \
-        headers.get(content_type_header.lower()) or \
-        headers.get(content_type_header.upper())
+    for key in headers:
```

Comment:
Could you please also add a test case for when there is no `Content-Type` header, and assert that the parser raises `ParseFailedException` in that case?

I think that test case might fail, since in this `for` loop, `content_type` is only set if it's found. If not found, the next `if content_type` would probably raise `NameError`.